### ISSUE TITLE
The operator 'aten::_ctc_loss' is not currently implemented for the MPS device

### ISSUE THREAD

**Issue by dalarconics** [pytorch/pytorch#160829]
### 🐛 Describe the bug

import torch
import torch.nn as nn

# Calcular la pérdida
# Pasamos los logits, las etiquetas, y las longitudes de ambas secuencias.
loss = ctc_loss(log_probs, targets, input_lengths, target_lengths)

### Versions

Collecting environment information...
PyTorch version: 2.7.1
Is debug build: False
CUDA used to build PyTorch: None
ROCM used to build PyTorch: N/A

OS: macOS 15.6 (arm64)
GCC version: Could not collect
Clang version: 17.0.0 (clang-1700.0.13.5)
CMake version: version 3.26.3
Libc version: N/A

Python version: 3.13.5 | packaged by Anaconda, Inc. | (main, Jun 12 2025, 11:23:37) [Clang 14.0.6 ] (64-bit runtime)
Python platform: macOS-15.6-arm64-arm-64bit-Mach-O
Is CUDA available: False
CUDA runtime version: No CUDA
CUDA_MODULE_LOADING set to: N/A
GPU models and configuration: No CUDA
Nvidia driver version: No CUDA
cuDNN version: No CUDA
Is XPU available: False
HIP runtime version: N/A
MIOpen runtime version: N/A
Is XNNPACK available: True

CPU:
Apple M4 Max

Versions of relevant libraries:
[pip3] numpy==2.3.1
[pip3] onnxruntime==1.22.1
[pip3] optree==0.17.0
[pip3] torch==2.7.1
[pip3] torchaudio==2.7.1
[pip3] torchvision==0.22.1
[conda] numpy                                       2.3.1            pypi_0              pypi
[conda] optree                                      0.17.0           pypi_0              pypi
[conda] torch                                       2.7.1            pypi_0              pypi
[conda] torchaudio                                  2.7.1            pypi_0              pypi
[conda] torchvision                                 0.22.1           pypi_0              pypi
